main.js

```diff
@@ -187,6 +187,7 @@ function insertErrorMessage (input, errorCont, message) {
 }
 
 function generateRandomNumber(min=0, max=100) {
+  timeStart = Date.now();
```

Comment:
Adding this line here violates single responsibility. I think a better place to put it would be the setRange function where it can, potentially, be broken out into another reset game function or something similar.